src/lib.rs

```diff
@@ -108,6 +109,10 @@ impl Command {
                 7,
             ))),
             [b'E', ..] => Ok(Some((Command::Bootload, 1))),
+            [b'F', target, msb, lsb, ..] => {
+                let value = u16::from_be_bytes([msb, lsb]);
+                Ok(Some((Command::FanSpeed { target, value }, 4)))
+            },
             [header, ..] if b"ABCD".contains(&header) => Ok(None),
```

Comment:
Should this condition be now extended to
```suggestion
            [header, ..] if b"ABCDEF".contains(&header) => Ok(None),
```
?

Reply:
Hmmm good eye! If I understand correctly, this only matters for partial reads (so we don't error out while we're building up to a complete command). It's probably a good idea to add a clarifying comment, since it's not a very clear line otherwise. Let's wait for the conclusion of [this conversation](https://github.com/tonarino/portal/issues/2623) before continuing to improve on this though.

Reply:
Yeah, I think you're right. And sure, let's wait. .)